financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/test/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/admin/DefaultConsentAdminHandlerTest.java

```diff
@@ -54,95 +67,168 @@
-        doReturn(true).when(consentCoreServiceImpl).revokeConsentWithReason(anyString(), anyString(),
+        doReturn(true).when(consentCoreServiceMock).revokeConsentWithReason(anyString(), anyString(),
                 anyString(), anyString());
-        doReturn(Map.of("HistoryId", TestUtil.getSampleConsentHistoryResource())).when(consentCoreServiceImpl)
+        doReturn(Map.of("HistoryId", TestUtil.getSampleConsentHistoryResource())).when(consentCoreServiceMock)
                 .getConsentAmendmentHistoryData(anyString());
         ArrayList<ConsentStatusAuditRecord> auditRecords = new ArrayList<>();
         auditRecords.add(TestUtil.getSampleConsentStatusAuditRecord(TestConstants.SAMPLE_CONSENT_ID,
                 TestConstants.AUTHORISED_STATUS));
-        doReturn(auditRecords).when(consentCoreServiceImpl).getConsentStatusAuditRecords(any(ArrayList.class),
+        doReturn(auditRecords).when(consentCoreServiceMock).getConsentStatusAuditRecords(any(ArrayList.class),
                 anyInt(), anyInt());
-        doReturn(TestUtil.getSampleConsentFileObject(TestConstants.SAMPLE_CONSENT_FILE)).when(consentCoreServiceImpl)
+        doReturn(TestUtil.getSampleConsentFileObject(TestConstants.SAMPLE_CONSENT_FILE)).when(consentCoreServiceMock)
                 .getConsentFile(anyString());
+
+        configs = new HashMap<String, Object>();
+        configs.put(FinancialServicesConstants.MAX_INSTRUCTED_AMOUNT, "1000");
+
+        FinancialServicesConfigParser configParserMock = mock(FinancialServicesConfigParser.class);
+        doReturn(configs).when(configParserMock).getConfiguration();
+        configParser.when(FinancialServicesConfigParser::getInstance).thenReturn(configParserMock);
+
+        headers = new HashMap<>();
+
+        ConsentAdminBuilder consentAdminBuilder = Mockito.mock(ConsentAdminBuilder.class);
+        Mockito.when(consentAdminBuilder.getConsentAdminHandler()).thenReturn(defaultConsentAdminHandler);
+        ConsentExtensionExporter.setConsentAdminBuilder(consentAdminBuilder);
+
+        defaultConsentAdminHandler = new DefaultConsentAdminHandler();
+        adminHandler = new DefaultConsentAdminHandler();
+        setPrivateBoolean(adminHandler, "isExtensionsEnabled", true);
+        setPrivateBoolean(adminHandler, "isExternalPreConsentRevocationEnabled", true);
+        setPrivateBoolean(adminHandler, "isExternalPreConsentSearchEnabled", true);
+
     }
 
     @AfterClass
     public void tearDown() {
         // Closing the mockStatic after each test
         consentExtensionsDataHolder.close();
+        configParser.close();
     }
 
     @Test
     public void testHandleSearch() {
+        doReturn(getQueryParams()).when(consentAdminDataMock).getQueryParams();
 
-        ConsentAdminData consentAdminDataMock = mock(ConsentAdminData.class);
+        defaultConsentAdminHandler.handleSearch(consentAdminDataMock);
+        //verify(consentAdminDataMock).setResponseStatus(ResponseStatus.OK);
```

Comment:
Is there a reason to commenting out this?

Reply:
no,it's a mistake